### DISCUSSION TITLE
Feature Request: allow users to reveal password on login page

### DISCUSSION THREAD

**Discussion by ananbas** [apache/superset#30643] [Ideas]
Hi team,

Thanks for the great products,
one of my users asking wether Superset has an options to reveal password in the Login page.
![login](https://github.com/user-attachments/assets/a96164c9-ee5f-4e5e-b9ba-82408320428a)

Thanks,
Ananbas

**Comment by hermanwibowo** [comment]
nice idea!

**Comment by sfirke** [comment]
Hi @ananbas, I have re-titled this to more clearly mark it as a feature request.  Superset is built on top of [Flask-AppBuilder](https://github.com/dpgaspar/Flask-AppBuilder) ("FAB") and uses its security module for authentication.  If anyone wants to implement this, it might need to be done upstream in FAB to get pulled into Superset.
